### PR TITLE
containerized-build: adjust defaults for 3.0

### PR DESCRIPTION
**Pull request by mfrw**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Adjust default values for 3.0

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- NA

###### Links to CVEs  
- NA

###### Test Methodology

- NA

### CODE REVIEW

File: toolkit/scripts/containerized-build/create_container_build.sh [hunk 1]
```diff
@@ -32,7 +32,7 @@ Optional arguments:
     MODE            build or test. default:"build"
                         In 'test' mode it will use a pre-built mariner chroot image.
                         In 'build' mode it will use the latest published container.
-    VERISION        1.0 or 2.0. default: "2.0"
+    VERISION        2.0 or 3.0. default: "3.0"
```
Comment (reubeno) [inline toolkit/scripts/containerized-build/create_container_build.sh:35]: While you're touching this, would you mind fixing the typo in `VERISION`; should be `VERSION` without the first `I`.

Comment (mfrw) [reply toolkit/scripts/containerized-build/create_container_build.sh:35]: Ofcourse sir

File: toolkit/scripts/containerized-build/create_container_build.sh [hunk 2]
```diff
@@ -228,7 +228,8 @@ if [[ "${mode}" == "build" ]]; then # Configure base image
         docker import "${chroot_file}" $container_img
     fi
 else
-    container_img="mcr.microsoft.com/cbl-mariner/base/core:${version}"
+    # Till we publish 3.0 image, use the image created by the build phase
+    container_img="mcr.microsoft.com/cbl-mariner/${USER}-containerized-rpmbuild:${version}"
```
Comment (neha170) [inline toolkit/scripts/containerized-build/create_container_build.sh:232]: This image would not be available until it has been run for build phase. We can get rid of the if-else here, and build the same image for both modes. We would need worker_chroot.tar.gz as well, so please remove the if-else from building worker_chroot.tar.gz, thanks

**Review by neha170 — COMMENTED** [review]
Thanks for looking out and making changes for 3.0 :)

**Comment by neha170** [comment]
Closing in favour of https://github.com/microsoft/CBL-Mariner/pull/7352 that contains these changes, and some others to make 3.0 work for containerized-rpmbuild

**Pull request closed without merging**